versioned/spi/src/main/java/org/projectnessie/versioned/ResultType.java

```diff
@@ -21,5 +21,6 @@ public enum ResultType {
   REFERENCE_CREATED,
   COMMIT,
   MERGE,
-  TRANSPLANT
+  TRANSPLANT,
+  CONTENT_RESULT
```

Comment:
I'd note that the other constants don't have the `_RESULT` suffix... but I'm OK to leave it as is.